code-studio/src/js/progress.js

```diff
@@ -56,9 +64,19 @@ progress.populateProgress = function (scriptName, puzzlePage) {
     // Merge progress from server (loaded via AJAX)
     var serverProgress = data.levels || {};
     Object.keys(serverProgress).forEach(function (levelId) {
-      // Only the server can speak to whether a level is submitted.  If it is,
-      // we show the submitted styling.
-      if (serverProgress[levelId].submitted) {
+      // Only the server can speak to whether a level is submitted/accepted/rejected.  If it is,
```

Comment:
They will be yellow ("in progress") because that is the highest level of progress the client is allowed to write down locally.

In reality these level types should never be included in a high-scale script because it doesn't make sense to write `FreeResponse` answers and submit them unless there is a server present.